uber/models.py

```diff
@@ -1135,10 +1135,6 @@ def shirt_size_marked(self):
         return self.shirt not in [c.NO_SHIRT, c.SIZE_UNKNOWN]
 
     @property
-    def is_group_leader(self):
```

Comment:
This was an unrelated fix. The exact same function is also just above `shirt_size_marked`, so this was a duplicate.